spec/metrics-spec.js

```diff
@@ -142,13 +173,14 @@ describe('Metrics', async () => {
       })
     })
 
-    describe('when the user is chosen to send commands', async () => {
+    describe('when shouldIncludePanesAndCommands is true', async () => {
       beforeEach(async () => {
         global.localStorage.setItem('metrics.userId', 'd')
         await atom.packages.activatePackage('metrics')
 
         const {mainModule} = atom.packages.getLoadedPackage('metrics')
         mainModule.shouldIncludePanesAndCommands = true
+        Reporter.addCustomEvent.reset()
```

Comment:
Given that we're [calling this in the `afterEach` function](https://github.com/atom/metrics/blob/a9afa070ff8b1d82815012f9a1da0c4c369a86b0/spec/metrics-spec.js#L34) for all tests in this file, can we remove this line, or do we need it?

Reply:
💥

Reply:
Removed the reset from all tests in this file, as it's unnecessary, and left it here.